.jenkins/pace_cache_generation.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -e
+
+if (( $# < 3 )); then
+    echo "USAGE: .jenkins/pace_cache_generation.sh backend experiment [target_uri]"
+fi
+
+backend=$1
+experiment=$2
+
+# If the target_uri starts with "daint:" then the Jenkins scripts are used
+# Use the target_uri as the last command line argument if passed, else the default daint path
+default_target_uri="daint:/scratch/snx3000/olifu/jenkins/scratch/store_gt_caches/$experiment/${backend//:/_}"
+target_uri=${1:-$daint_target_uri}
+
+if [[ $target_uri == daint:* ]]; then
+    use_jenkins_action=true
+    target_dir=${target_uri#daint:}
+else
+    use_jenkins_action=false
+    target_dir=$target_uri
+fi
+
+script_dir="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+pace_dir=$script_dir/../
+[[ "${NODE_NAME}" == *"daint"* ]] && source ~/.bashrc
+
+export LONG_EXECUTION=1
+
+if [[ $use_jenkins_action == "true" ]]; then
+    .jenkins/jenkins.sh initialize_driver $backend $experiment
+else
+    EXPERIMENT=$experiment TARGET=driver make get_test_data
+    $pace_dir/.jenkins/initialize_driver.py test_data/8.1.0/$experiment/driver $backend
```

Comment:
If `make get_test_data` outputs the version then we don't need to duplicate that logic here.